src/main/java/com/jland/website/service/impl/PresentationServiceDefault.java

```diff
@@ -0,0 +1,23 @@
+package com.jland.website.service.impl;
+
+import com.jland.website.model.PresentationPlanItem;
+import com.jland.website.repository.PresentationRepository;
+import com.jland.website.service.PresentationService;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class PresentationServiceDefault implements PresentationService {
```

Comment:
why default intead of Impl based on other services

Reply:
renamed to Impl